applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -605,6 +614,41 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information
 
 .. _matter_bridge_testing:
 
+Configure the functionality of the Matter-Bridge device
+-------------------------------------------------------
+
+The device supports two configurations: basic and extended. 
```

Comment:
I'm not sure whether we should describe basic configuration at all, as we have the commands for it in the Building and running section. Also I would recommend naming this "extended" configuration directly like "Configure the Matter smart plug functionality" or sth, as I don't think that using mystery extended name is necessary.

Reply:
This was not resolved.